jamovi/surv.r.yaml

```diff
@@ -44,120 +64,86 @@ items:
           title: ''
           type: text
           content: Tarone-Ware
           visible: (tests:tarone-ware)
-
         - name: stat[peto-peto]
           title: ''
           type: text
           content: Peto-Peto
           visible: (tests:peto-peto)
-
-        - name: nu[logrank]
-          title: ν
-          visible: (tests:logrank)
-
-        - name: nu[gehan]
-          title: ν
-          visible: (tests:gehan)
-
-        - name: nu[tarone-ware]
-          title: ν
-          visible: (tests:tarone-ware)
-
-        - name: nu[peto-peto]
-          title: ν
-          visible: (tests:peto-peto)
-
-        - name: nuse[logrank]
-          title: SE
+        - name: chisqr[logrank]
+          title: Chi-Squared
+          type: number
           visible: (tests:logrank)
-
-        - name: nuse[gehan]
-          title: SE
+        - name: chisqr[gehan]
+          title: Chi-Squared
+          type: number
           visible: (tests:gehan)
-
-        - name: nuse[tarone-ware]
-          title: SE
+        - name: chisqr[tarone-ware]
+          title: Chi-Squared
+          type: number
           visible: (tests:tarone-ware)
-
-        - name: nuse[peto-peto]
-          title: SE
+        - name: chisqr[peto-peto]
+          title: Chi-Squared
+          type: number
           visible: (tests:peto-peto)
-
-        - name: z[logrank]
-          title: z
-          format: zto,p
+        - name: df[logrank]
+          title: DF
+          type: integer
           visible: (tests:logrank)
-
-        - name: z[gehan]
-          title: z
-          format: zto,p
+        - name: df[gehan]
+          title: DF
+          type: integer
           visible: (tests:gehan)
-
-        - name: z[tarone-ware]
-          title: z
+        - name: df[tarone-ware]
+          title: DF
+          type: integer
           visible: (tests:tarone-ware)
-
-        - name: z[peto-peto]
-          title: z
+        - name: df[peto-peto]
+          title: DF
+          type: integer
           visible: (tests:peto-peto)
-
-        - name: p[logrank]
-          title: p
+        - name: pvalue[logrank]
+          title: P-Value
           type: number
           format: zto,pvalue
           visible: (tests:logrank)
-
-        - name: p[gehan]
-          title: p
+        - name: pvalue[gehan]
+          title: P-Value
           type: number
           format: zto,pvalue
           visible: (tests:gehan)
-
-        - name: p[tarone-ware]
-          title: p
+        - name: pvalue[tarone-ware]
+          title: P-Value
           type: number
           format: zto,pvalue
           visible: (tests:tarone-ware)
-
-        - name: p[peto-peto]
-          title: p
+        - name: pvalue[peto-peto]
+          title: P-Value
           type: number
           format: zto,pvalue
           visible: (tests:peto-peto)
 
+
     - name: sc
       title: Survival Curve
       type: Image
       visible: (sc)
       width: 600
-      height: 400
+      height: 600
```

Comment:
i prefer the look of 600x400, but i'm happy with 600 high if you think it's better

Reply:
The reason I did this is because of the risk table (which requires space at the bottom). If there are many levels for the grouping variable the table runs out of space. Not sure if there is a way to change the height of the the plot dynamically (depending on whether the risk table option is checked) but my guess is that it wouldn't be easy. 
I'll keep it as it is for the moment.

Reply:
inside `.init()` you can go:

```{r}
if (self$options$risktable)
    self$results$plot$setSize(600, 600)  # at least i think you can
```

Reply:
This is working now :-) Thanks for the help !